generators/app/templates/common/pyproject.toml

```diff
@@ -90,6 +90,8 @@ ban-relative-imports = "all"
 
 [tool.ruff.flake8-type-checking]
 strict = true
+runtime-evaluated-base-classes = ["pydantic.BaseModel"]
+# Pydantic need to be able to evaluate types at runtime; see https://pypi.org/project/flake8-type-checking/
```

Comment:
- need ⇒ needs
- wouldn't be clearer to link https://beta.ruff.rs/docs/settings/#flake8-type-checking-runtime-evaluated-base-classes instead? because configuration is different for `flake8-type-checking` and `ruff` rules

Reply:
The problem is the Ruff documentation does not really explain why this parameter exists and which modules are concerned, unlike the Flake8 one which is more explicit. Maybe I can link both of them ?

Reply:
Yes, let's link both of them!